app/models/decidim/consultations/vote.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Consultations
+    # The data store for question's votes in the Decidim::Consultations component.
+    class Vote < ApplicationRecord
+      belongs_to :question,
+                 foreign_key: "decidim_consultation_question_id",
+                 class_name: "Decidim::Consultations::Question",
+                 counter_cache: :votes_count,
+                 inverse_of: :votes
+
+      belongs_to :author,
```

Comment:
Why not use the `Authorable` concern in `core`?

Reply:
Is there a way to use authorable concern without having the author optional?

Reply:
So you want the author to be obligatory? What about adding the authorable and then add the validations in the model? 🤔  This would enable you to have user groups as authors too, but I don't know if this is a wanted feature.

Reply:
In the consultations context I'm assuming that votes are only available for individuals, since interested entities will be managed with a different engine. Anyway we've the sprint meeting today. We wil disscuss it with @xabier later and take a decission about this